Frontend/Dockerfile

```diff
@@ -16,13 +16,13 @@ COPY nginx.conf.template .
 RUN envsubst '$REACT_APP_BACKEND_URL,$SITEMAP_LOCATION' < ./nginx.conf.template > ./nginx.conf
 
 # Install yarn
-RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
-RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" > /etc/apt/sources.list.d/yarn.list
-RUN apt-get update && apt-get install -y yarn
+#RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
+#RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" > /etc/apt/sources.list.d/yarn.list
+#RUN apt-get update && apt-get install -y yarn
```

Comment:
Don't comment out code. If it's no longer needed, delete it.

Reply:
I removed it. Sorry about that, I forgot to remove those lines while I was trying to get everything working.